corporate/lib/stripe.py

```diff
@@ -1169,6 +1207,65 @@ def set_required_plan_tier(self, required_plan_tier: int) -> str:
             plan_tier_name = CustomerPlan.name_from_tier(new_plan_tier)
         return f"Required plan tier for {self.billing_entity_display_name} set to {plan_tier_name}."
 
+    def configure_fixed_price_plan(self, fixed_price: int) -> str:
+        customer = self.get_customer()
+        if customer is None:
+            customer = self.update_or_create_customer()
+
+        if customer.required_plan_tier is None:
+            raise SupportRequestError("Required plan tier should not be set to None")
+        required_plan_tier_name = CustomerPlan.name_from_tier(customer.required_plan_tier)
+
+        fixed_price_cents = fixed_price * 100
+        fixed_price_plan_params: Dict[str, Any] = {
+            "fixed_price": fixed_price_cents,
+            "tier": customer.required_plan_tier,
+            "status": CustomerPlan.NEVER_STARTED,
```

Comment:
This shoudl move into the if statement, since it's overwritten for the offer case.

Reply:
#28749 -- commit 1